src/components/SingleEvent/EventInfo.js

```diff
@@ -144,18 +146,25 @@ class EventInfo extends Component {
         </Location>
         <TotalPot>
           <EthIcon />
-          Total pot{' '}
+          Total pot:{' '}
           <span>
             {toEthVal(party.deposit)
               .mul(party.participants.length)
               .toEth()
               .toFixed(2)}{' '}
             ETH
           </span>
-          RSVP{' '}
+          RSVP:{' '}
           <span>
             <DepositValue value={party.deposit} /> ETH
           </span>
+          <span>
+            <strong>Cooling Period: </strong>
```

Comment:
Why is only cooling period in strong/bold while others not?

Reply:
And would be nice if you can link to https://kickback.events/faq (target=blank) as we do in signup page

Reply:
Ok, I can sign off once the link is added